index.js

```diff
@@ -321,7 +321,7 @@ class GoogleAuth extends DbBase {
     const admin = await this._getAdmin(sentEmail)
 
     const isValidPassword = this.conf.useDB
-      ? (await verify(sentPassword, (admin?.password || '')))
+      ? (await verify((sentPassword || ''), (admin?.password || '')))
```

Comment:
Not fully confortable with this. 
For example if admin.password is false and the sentPassword is '' might get verified. 
What about doing: 
```console
? sentPassword && admin?.password && await verify(sentPassword, admin.password)
```